packages/firebase_database/test/firebase_database_test.dart

```diff
@@ -117,6 +117,16 @@ void main() {
           ]),
         );
       });
+      test('update', () async {
+        final dynamic value = <String, dynamic>{'hello': 'world'};
+        await database.reference().child("foo").update(value);
+        expect(
+            log,
+            equals(<MethodCall>[
+              new MethodCall("DatabaseReference#update",
+                  <String, dynamic>{'path': 'foo', 'value': value})
```

Comment:
Can you put `"DatabaseReference#update"` on its own line and put a trailing comma at the end of `}` to avoid the four-space intent here?